activesupport/lib/active_support/core_ext/numeric/conversions.rb

```diff
@@ -134,6 +134,12 @@ def to_formatted_s(*args)
   deprecate to_formatted_s: :to_s
 end
 
-[Fixnum, Bignum, Float, BigDecimal].each do |klass|
-  klass.prepend(ActiveSupport::NumericWithFormat)
+# Ruby 2.4+ unifies Fixnum & Bignum into Integer.
+if Integer == Fixnum
```

Comment:
Didn't the feature mention warning when `Fixnum`is directly referenced? Should we check for the ruby version explicitly instead?